plugins/gatsby-plugin-auto-index-pages/gatsby-node.js

```diff
@@ -130,6 +131,22 @@ exports.createPages = async ({ actions, graphql, reporter }, pluginOptions) => {
       },
     });
 
+    const redirectsFrom = taxonomyRedirects.find(({ url }) => url === slug);
+
+    if (redirectsFrom) {
+      console.log(`Setup taxonomy redirects for ${slug}`);
+
+      redirectsFrom.paths.forEach((from) => {
+        console.log(`\tRedirect from ${from}`);
```

Comment:
This will likely get pretty noisy, pretty fast. If we value the logging, could we instead use [`reporter.verbose`](https://www.gatsbyjs.com/docs/reference/config-files/node-api-helpers/#reporter) so that these only show up if you pass a `--verbose` flag when running `gatsby develop` or `gatsby build`?

Reply:
Oh good call. I had this in for debugging and was on the fence about taking it out.

I might just strip it out, since this will almost always be the same results every time.

Reply:
Actually, after looking at `reporter.verbose` I changed my mind. That looks pretty handy.